napari_clusters_plotter/_utilities.py

```diff
@@ -327,9 +336,7 @@ def generate_cluster_image(label_image, label_list, predictionlist):
     """
 
     predictionlist_new = np.array(predictionlist) + 1
-    plist = np.zeros(
```

Comment:
Does reverting this change drastically change the time to execute the function? Otherwise I actually prefer a more robust function even if we don't specifically make use of this use-case anymore. If it is adding a lot of processing time then of course we can change it

Reply:
There is some speedup, but no drastic speedup when reverting this.
I would still suggest to remove/revert it, since this would reduce code complexity (think of our future selves or even on newbie contributors). 
Regarding robustness: we have some more tests now that would cover non-robust changes of this in the future.

Reply:
I guess it does reduce code complexity but in the case other people are using this function in place of the old one which uses the mapping it might be better to have a function that behaves as before. I will add some comments to the function to make clear why the extra line is there